### ISSUE TITLE
setup missing 'vivarium_cluster_tools'

### ISSUE THREAD

**Issue by aflaxman** [ihmeuw/vivarium_csu_swissre_cervical_cancer#8]
Here is an error from trying to follow the README instructions to get started with this repo:

```
$ make_specs -v
...
File "/ihme/homes/abie/projects/2020/vivarium_csu_swissre_cervical_cancer/src/vivarium_csu_swissre_cervical_cancer/tools/make_artifacts.py", line 16, in <module>
    import vivarium_cluster_tools as vct
ModuleNotFoundError: No module named 'vivarium_cluster_tools'
```

`pip install vivarium_cluster_tools` seems to have fixed it.

**Comment by mattkappel** [comment]
Hmm. This project has vivarium_cluster_tools as an 'extra' module that gets installed on `pip install -e .[dev]`, which is not in the README.  This is the same behavior as in breast cancer, but perhaps this ought to change? That said vivarium_cluster_tools are unnecessary if not running on the IHME cluster.

**Comment by aflaxman** [comment]
Maybe some of the functions in `make_artifacts.py` can work without it; when it comes to required modules, less is more.

**Comment by mattkappel** [comment]
I agree, and it doesn't look like `vivarium_cluster_tools` is doing any heavy lifting for `make_artifacts.py` -- it's just using `vivarium_cluster_tools`'s mkdir method, which we can do another way.
